src/formic/utils/lapack_interface.cpp

```diff
@@ -76,6 +76,8 @@ double formic::ddot(int n, const double *x, int incx, const double *y, int incy)
 double formic::xdot(int n, const double *x, int incx, const double *y, int incy) {
   return ::FC_GLOBAL(ddot,DDOT)(&n, x, &incx, y, &incy);
 }
+
+/* commented to avoid compiler error.
```

Comment:
Note to future devs: *please* write sufficiently meaningful comments. Which compiler? Which version? Which options?